### PR TITLE
Promise module result classes

### PR DESCRIPTION
**Pull request by vpodzime**
Support for result classes from promise modules.

### CODE REVIEW

**Comment by vpodzime** [comment]
@cf-bottom jenkins, please

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/5979/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/5979/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-5979/

File: libpromises/mod_custom.c [hunk 1]
```diff
@@ -673,7 +673,8 @@ static PromiseResult PromiseModule_Evaluate(
     PromiseResult result;
     const char *const result_str = JsonObjectGetAsString(response, "result");
 
-    Attributes a = GetClassContextAttributes(ctx, pp); // TODO: WTF
+    /* Attributes needed for setting outcome classes etc. */
+    Attributes a = GetClassContextAttributes(ctx, pp);
```
Comment (olehermanse) [inline libpromises/mod_custom.c:715]: 😆 👍

**Review by olehermanse — APPROVED** [review]
👍

**Pull request merged**